dev/com.ibm.websphere.javaee.jaxb.2.3/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.websphere.javaee.jaxb.2.3; singleton:=true
+
+Export-Package: javax.xml.bind.*;version="2.3"
+
+Private-Package: org.apache.geronimo.osgi.locator
+
+Import-Package: \
+  !javax.activation, \
```

Comment:
dynamic import of javax.activation can be replaced with a regular import, since this feature will always resolve with the javax.activation bundle